src/handlers/testing/exec/util/ci.ts

```diff
@@ -92,14 +84,25 @@ function repositoryFromEvent(event: { repository?: unknown }): Repository {
   return zRepositorySchema.parse(event.repository);
 }
 
-function workflowInputsFromEvent(event: {
-  inputs?: unknown;
-}): WorkflowInputs | null {
-  const parsed = zWorkflowInputsSchema.safeParse(event.inputs);
-  if (parsed.success) {
-    return parsed.data;
+/**
+ * For users with UI-triggered test runs configured, they'll
+ * have their GitHub Actions workflow set up to receive the
+ * prompt snapshots as an input. This function is used to
+ * parse that input out of the event.json file, which will
+ * contain the workflow event payload.
```

Comment:
Understanding check - 

Is a prompt snapshot the resulting JSON from generating some version of a prompt with Autoblocks?

Reply:
it is the ID of the prompt snapshot we create on our end, see https://github.com/autoblocksai/autoblocks/pull/1263

Reply:
there is an example payload in the PR description:

```
gh workflow run e2e.yml \
  --ref nicole/epd-719-cli-prompt-snapshot-id-prompt-snapshot-ids \
  --field prompt-snapshots='{"text-summarization":"123","text-classififer":"456"}'
```

Reply:
Ok makes sense 👍